components/Cards/FlatCard/FlatCard.js

```diff
@@ -4,28 +4,35 @@ import classNames from 'classnames';
 import styles from './FlatCard.css';
 
 FlatCard.propTypes = {
+  button: PropTypes.element,
   children: PropTypes.node.isRequired,
   className: PropTypes.string,
   header: PropTypes.node,
-  imageAlt: PropTypes.string.isRequired,
-  imageSource: PropTypes.string.isRequired,
+  imageAlt: PropTypes.string,
+  imageSource: PropTypes.string,
 };
 
 FlatCard.defaultProps = {
+  button: null,
   className: undefined,
   header: undefined,
+  imageAlt: undefined,
+  imageSource: undefined,
 };
 
-function FlatCard({ children, className, header, imageSource, imageAlt }) {
+function FlatCard({ button: Button, children, className, header, imageSource, imageAlt }) {
   return (
     <article className={classNames(styles.FlatCard, className)}>
       <div className={styles.borderContainer}>
         {header && <div className={styles.header}>{header}</div>}
-        <div className={styles.rowCenter}>
-          <img className={styles.image} src={imageSource} alt={imageAlt} />
-        </div>
+        {imageSource && (
+          <div className={styles.rowCenter}>
+            <img className={styles.image} src={imageSource} alt={imageAlt} />
+          </div>
+        )}
         {header && <hr className={styles.divider} />}
         <div className={styles.children}>{children}</div>
+        {Button}
```

Comment:
I thought you would put the button into a div and then provide an opinionated positioning of any/all buttons that come into place?

Reply:
Sorry, just re-read convo from 2/26 in Slack and saw you mentioned opinionated positioning. To make sure we're on same page: parent div to button and a prop with choices `left`, `center`, `right` for positioning?

Reply:
no choices for positioning; always center.